index.bs

```diff
@@ -1307,6 +1313,29 @@ example of the latter, when the user is accessing the [=[RP]=] from a given clie
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
 
+### Attestation Preference enumeration (enum <dfn enum>AttestationPreference</dfn>) ### {#attestation}
+
+[=[RPS]=] may use the {{AttestationPreference}} enum to specify their preference regarding attestation types used 
+during credential generation.
+
+<pre class="idl">
+    enum AttestationPreference {
+        "low-cost",
+        "verifiable" 
+    };
+</pre>
+
+<ul>
+    <li><dfn>low-cost</dfn> - indicates that the [=[RP]=] prefers a low-cost attestation type. For example, if a user agent
+      supports self attestion and Privacy CA, the user agent SHOULD use self attestation, avoiding calls to the Privacy
+      CA, user consent UI, etc. This is the default value.
```

Comment:
I do not think this should be the default. I think we should encourage RPs to be secure by default and insecure in well thought out extraordinary situations.

Reply:
Users will be plenty secure with software-only authenticators (i.e., those that don't have meaningful attestations). Also, the other cases likely mean that the client will prompt the user with consent UI that the user may or may not consent to. Users aren't secure if they don't allow their authenticators to be used because of privacy concerns. With this being the default, we don't have to worry that users will opt out, and we give them great protections.

Reply:
Yes, you're right. I've changed my mind since writing the previous comment. I agree that no attestation is a reasonable default.